proto/one/common/schedule/schedule.proto

```diff
@@ -7,22 +7,35 @@ import "record_auditinfo.proto";
 option csharp_namespace ="ONE.Models.CSharp";
 
 message Schedule {
+    
+    // unique identifier of the schedule
     string id = 1;
 
+    // schedule name
     string name = 2;
 
-    google.protobuf.StringValue twinRefId = 3;
+    //schedule description
+    google.protobuf.StringValue Description = 3;

```

Comment:
field names should be camelCased, also below on line 21

Reply:
I changed the names.